crates/nargo/src/resolver.rs

```diff
@@ -79,12 +79,12 @@ impl<'a> Resolver<'a> {
         let (entry_path, crate_type) = super::lib_or_bin(dir_path)?;
 
         let manifest_path = super::find_package_manifest(dir_path)?;
-        let manifest = super::manifest::parse(manifest_path)?;
+        let manifest = super::manifest::parse(&manifest_path)?;
 
         let crate_id = driver.create_local_crate(entry_path, crate_type);
 
         let mut resolver = Resolver::with_driver(&mut driver);
-        resolver.resolve_manifest(crate_id, manifest)?;
+        resolver.resolve_manifest(crate_id, manifest, manifest_path.parent().unwrap())?;
```

Comment:
can you change unwrap to expect with a message stating why its okay to unwrap?
Something like `.expect("every manifest path has a parent")`

Reply:
I'd probably want to move this onto its own line rather than doing this inline with the call to `resolve_manifest` however.